.github/workflows/docs.yml

```diff
@@ -0,0 +1,100 @@
+name: Documentation
+
+# Controls when the workflow will run.
+on:
+
+  # Allow the workflow to be reusable.
+  workflow_call:
+    inputs:
+      docs_path:
+        description: Path to the documentation directory, relative to the repository root.
+        required: false
+        default: .
+        type: string
+      system_dependencies:
+        description: Optional list of system dependencies to install via `apt` command, separated by spaces.
+        required: false
+        default: gettext
+        type: string
+      python_version:
+        description: The version of Python binary to use.
+        required: false
+        default: '3.7'
+        type: string
+      requirements_path:
+        description: Path to the `requirements.txt`, relative to the repository root.
+        required: false
+        default: requirements.txt
+        type: string
+
+jobs:
+  build:
+    name: Build
```

Comment:
Maybe `Lint & Build` or `Test & Build` or similar?

Reply:
I tried all of those, Github will ellipsize it anyway in most views. Better to be concise in this case and we can improve it later if we have additional jobs.